### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by trobblob**
Some text colors on Dynamics CRM websites stay black when turning on dark mode, so I fixed that.

### CODE REVIEW

**Comment by alexanderby** [comment]
Hi, did you check that it works?

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -99,6 +99,15 @@ INVERT
 
 ================================
 
+*.crm*.dynamics.com
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:102]: Does this URL pattern work?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -99,6 +99,15 @@ INVERT
...
+
+CSS
+.ms-Label {
+    color: var(--dark-neutral-text);
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:106]: Maybe it should look like
```
color: var(--darkreader-neutral-text) !important;
```

**Review by alexanderby — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (alexanderby) [reply src/config/dynamic-theme-fixes.config:106]: Made an Approve by accident

**Review by alexanderby — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (trobblob) [reply src/config/dynamic-theme-fixes.config:102]: This URL pattern worked when I tested it with the dark reader dev tools.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (trobblob) [reply src/config/dynamic-theme-fixes.config:106]: I had no problem without `!important`. But, if it's better to use `!important`, I can make that change.

Comment (alexanderby) [reply src/config/dynamic-theme-fixes.config:106]: Please update the variable and the !important, check if it works and I will fix the conflict (probably similar lines were edited in another Pull Request by someone else).

**Review by alexanderby — APPROVED** [review]

**Pull request merged**